docs/contributing/faq.md

```diff
@@ -26,9 +27,7 @@ We have a large backlog of pull requests to get through and the team are moving
 
 The number one factor we look at when deciding what issues to look at are your 👍 [reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue/PR description as these can be [easily discovered](https://github.com/hashicorp/terraform-provider-aws/issues?q=is%3Aopen+sort%3Areactions-%2B1-desc). Comments that further explain desired use cases or poor user experience are also heavily factored. The items with the most support are always on our radar, and we commit to keep the community updated on their status and potential timelines.
 
-Once we have prioritized your contribution for review, we will let you know when to expect an engineer to get in touch. If changes are required, we will ask in the pull request. If you are unable to find the time, just let us know, and we will make the necessary changes required in order to merge.
-
-We publish a [road map](../../ROADMAP.md) every quarter which describes major themes or specific product areas of focus.
+We publish a [road map](../../ROADMAP.md) every quarter which describes major themes or specific product areas of focus. What is excluded from the public roadmap is work performed under NDA with AWS on new services, and any ad-hoc work we pick up during the quarter. This ad-hoc work can be responding to bugs, gardening day activity, customer prioritization, and technical debt items.
```

Comment:
`road map` or `roadmap`? Both are used in this paragraph.

Reply:
Elsewhere, it seems like we usually use `roadmap`, so I'll get that corrected 🙂